### PR TITLE
Added text to Initial I/O pin configuration section of EBB command do…

### PR DESCRIPTION
**Pull request by EmbeddedMan**
…cumentation describing Port B pins and if they are 5V tolerant or not.

Fixes issue #161

### CODE REVIEW

File: docs/ebb.html [hunk 1]
```diff
@@ -2677,6 +2677,17 @@ <h2><a name="states"></a>Initial I/O pin configuration</h2>
         and defaults to an off state at reset; see the <code><a href="#SR">SR</a></code> command.
         </p>
         
+        <p>Pins B1, B0, B2 and B3 are not 5-volt tolerant and any voltage above about 3.6V will damage them.
+        Pins B4, B5, B6 and B7 are 5-volt tolerant and will not be damaged by voltages up to 6.0V.
+        </p>
+
+        <p>Because all Port B pins (B0 through B7) have weak pull up resistors, any of these pins can easily be
```
Comment (oskay) [inline docs/ebb.html:2684]: Strike "easily" and "simply"

File: docs/ebb.html [hunk 2]
```diff
@@ -2707,54 +2718,63 @@ <h2><a name="states"></a>Initial I/O pin configuration</h2>
               <td>Pin</td>
               <td>Default Direction</td>
               <td>Default State</td>
+              <td>5V Tolerant?</td>
```
Comment (oskay) [inline docs/ebb.html:2721]: Drop question mark

File: docs/ebb.html [hunk 3]
```diff
@@ -2677,6 +2677,17 @@ <h2><a name="states"></a>Initial I/O pin configuration</h2>
         and defaults to an off state at reset; see the <code><a href="#SR">SR</a></code> command.
         </p>
         
+        <p>Pins B1, B0, B2 and B3 are not 5-volt tolerant and any voltage above about 3.6V will damage them.
+        Pins B4, B5, B6 and B7 are 5-volt tolerant and will not be damaged by voltages up to 6.0V.
```
Comment (oskay) [inline docs/ebb.html:2681]: IIRC, the datasheet says 5.5 V.

**Comment by oskay** [comment]
I've left a few minor comments. Otherwise, looks good to go ahead and merge.

File: docs/ebb.html (hunk 3 above)
Comment (EmbeddedMan) [reply docs/ebb.html:2681]: I did check the datasheet before putting these numbers in here. It has 6.0V listed as the abs max for any I/O pin with respect to Vss. Would you like me to de-rate this value down to 5.5V to provide a less risky upper end?

Comment (oskay) [reply docs/ebb.html:2681]: My datasheet does list 6.0 V in the Absolute Maximum table, but it describes them elsewhere as "5.5 V tolerant."

I guess that this is technically correct.

**Comment by EmbeddedMan** [comment]
The more I think about this the more I like your suggestion of 5.5V. If
somebody really cares about that half volt between 5.5V and 6.0V they'll
likely know to go to the datasheet themselves. I'll change it to 5.5V.

**Pull request merged**